tests/unit/moduleapi/propagate.tcl

```diff
@@ -761,3 +761,30 @@ tags "modules aof" {
     }
     }
 }
+
+test {Replicas that was marked as CLIENT_CLOSE_ASAP should not keep the replication backlog from been trimmed} {
+    start_server [list overrides [list loadmodule "$testmodule"]] {
+        set replica [srv 0 client]
+        start_server [list overrides [list loadmodule "$testmodule"]] {
+            set master [srv 0 client]
+            set master_host [srv 0 host]
+            set master_port [srv 0 port]
+            $master config set client-output-buffer-limit "replica 10mb 5mb 0"
+
+            # Start the replication process...
+            $replica replicaof $master_host $master_port
+            wait_for_sync $replica
+
+            test {module propagates from timer} {
+                # Replicate large commands to make the replica disconnected.
+                $master propagate-test.verbatim 100000 [string repeat "a" 1000] ;# almost 100mb
```

Comment:
by looking at the module code, it looks like you don't use the second argument, and the comment here is wrong.

Reply:
no, because RedisModule_ReplicateVerbatim() will replicate the whole client argv include `[string repeat "a" 1000]`, so that we can use a small command to replica a very large command.